Sources/AccessibilitySnapshotCore/AccessibilitySnapshotCore.h

```diff
@@ -0,0 +1,20 @@
+//
+//  AccessibilitySnapshotCore.h
+//  AccessibilitySnapshotCore
+//
+//  Created by Andrei Uriadov on 05.07.23.
+//
```

Comment:
Let's update this to the standard license header
```suggestion
//
//  Copyright 2023 Block Inc.
//
//  Licensed under the Apache License, Version 2.0 (the "License");
//  you may not use this file except in compliance with the License.
//  You may obtain a copy of the License at
//
//  http://www.apache.org/licenses/LICENSE-2.0
//
//  Unless required by applicable law or agreed to in writing, software
//  distributed under the License is distributed on an "AS IS" BASIS,
//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
//  See the License for the specific language governing permissions and
//  limitations under the License.
//
```